src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Ball.Colors getRandomColor() {
+        int index = new Random().nextInt(Ball.Colors.values().length);
```

Comment:
Creating a new instance of Random every time the method is called is inefficient. It's better to create a single instance as a class member and reuse it.